src/main/java/com/jfrog/ide/idea/scan/data/applications/JFrogApplicationsConfig.java

```diff
@@ -0,0 +1,25 @@
+package com.jfrog.ide.idea.scan.data.applications;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Getter
+@NoArgsConstructor
+public class JFrogApplicationsConfig {
+    @JsonProperty("version")
+    private String version;
+    @JsonProperty("modules")
+    private List<ModuleConfig> modules;
+
+    public JFrogApplicationsConfig(boolean createEmptyModules) {
+        if (createEmptyModules) {
```

Comment:
Since it is always true, please consider creating using a static creator:
```java
public static JFrogApplicationsConfig createApplicationConfigWithModule()
```
or, better, to add by default the source root to the default module:
```java
public static JFrogApplicationsConfig createApplicationConfigWithDefaultModule(Project project)
```